docker-compose.yml

```
@@ -25,6 +25,9 @@ services:
       - FIDESOPS__LOG_PII=${FIDESOPS__LOG_PII}
       - FIDESOPS__HOT_RELOAD=${FIDESOPS__HOT_RELOAD}
       - FIDESOPS__ROOT_USER__ANALYTICS_ID=${FIDESOPS__ROOT_USER__ANALYTICS_ID}
+      - VAULT_ADDR=${VAULT_ADDR}
+      - VAULT_NAMESPACE=${VAULT_NAMESPACE}
+      - VAULT_TOKEN=${VAULT_TOKEN}
```

Can you try testing locally with this change:
```suggestion
      - VAULT_ADDR
      - VAULT_NAMESPACE
      - VAULT_TOKEN
```
I don't believe the placeholder values should be necessary (for any of these ENV vars).

---

That worked 👍